ci/jenkins/test-vmc.sh

```diff
@@ -131,6 +146,43 @@ case $key in
     CODECOV_TOKEN="$2"
     shift 2
     ;;
+    --provider)
+    provider="$2"
+    shift 2
+    if [ "$provider" = "aws" ]; then
+      SSH_USERNAME=ubuntu
+      SKIP_LIST="TestEgress|TestProxy|TestProxyHairpinIPv4"
```

Comment:
May I know why these test cases need to be skipped? If need to skip them, maybe add short comment here to explain.